rs/universal_canister/lib/src/lib.rs

```diff
@@ -7,17 +7,15 @@
 
 pub mod management;
 
-use hex_literal::hex;
 use ic_types::Cycles;
 use universal_canister::Ops;
-/// The binary of the universal canister as compiled from
-/// `rs/universal_canister/impl`.
-///
-/// For steps on how to produce it, please see the README in
-/// `rs/universal_canister`.
-pub const UNIVERSAL_CANISTER_WASM: &[u8] = include_bytes!("universal-canister.wasm.gz");
-pub const UNIVERSAL_CANISTER_WASM_SHA256: [u8; 32] =
-    hex!("9c0b4ed1d729ffdd0e8d194df3be621f58a2fb86e1f1bbf556d89f43637de303");
+
+/// The binary of the universal canister as compiled from `rs/universal_canister/impl`.
+pub const UNIVERSAL_CANISTER_WASM: &[u8] = include_bytes!(env!("UNIVERSAL_CANISTER_WASM_PATH"));
```

Comment:
why not use a runtime `std::env` lookup? won't `env!` hardcode the path to something that might not exist at runtime anymore?

Reply:
I don't think a runtime `std::env` lookup works in combination with the `include_bytes!` macro invocation. We apparently haven't run into the issue on CI but I understand the concern. What we could do is add the following attributes to the `rust_library(name = "lib", ... )` call just to be sure: 

```
    rustc_env = {
        "UNIVERSAL_CANISTER_WASM_PATH": "$(location //rs/universal_canister/impl:universal_canister.wasm.gz)",
    },
    data = ["//rs/universal_canister/impl:universal_canister.wasm.gz"],
```

Reply:
I simplified it further by just not using the `build.rs` script in the bazel build and moving the setting of the env var to the `rust_library`.